README.md

```diff
@@ -72,4 +72,4 @@ They were written specifically for Android apps.
 
 ## Licensing
 
-Java Deobfuscator is licensed under the Apache 2.0 license.
\ No newline at end of file
+Java Deobfuscator is licensed under the Apache 2.0 license.
```

Comment:
What is this? Did you change anything?

Reply:
no, i don't know why this is happened. I didn't change anything at that line

Reply:
It doesn't matter really. Now you should force-push back to `a79729ed20b0f43da9b8095133d9f1cec21ed649`.